lib/game_teams.rb

```diff
@@ -19,7 +19,7 @@ def initialize(game_team_data)
         @tackles = game_team_data[:tackles].to_i
     end
 
-
-
-
+    def accuracy
+        @goals.to_f / @shots
+    end
```

Comment:
Thanks, this will be useful for the least and most accurate teams methods in stat tracker!!